ddl/ddl.go

```diff
@@ -212,7 +212,7 @@ type DDL interface {
 	CreateTable(ctx sessionctx.Context, stmt *ast.CreateTableStmt) error
 	CreateView(ctx sessionctx.Context, stmt *ast.CreateViewStmt) error
 	CreateTableWithLike(ctx sessionctx.Context, ident, referIdent ast.Ident, ifNotExists bool) error
-	DropTable(ctx sessionctx.Context, tableIdent ast.Ident) (err error)
+	DropTable(ctx sessionctx.Context, tableIdent ast.Ident, isView bool) (err error)
```

Comment:
Why not add a new interface named `DropView`?

Reply:
You can extract the common code of the dropping view and dropping table, and use diffrent API.

Reply:
Because I just reuse `*ast.DropTableStmt` to drop view in parser,, and 99% percent of two function action as same way. @shenli @winkyao  If you perfer add a new interface,I think we may only make some different in `DDL` package, we still need to use some code in `executor` package.
```go

	switch x := e.stmt.(type) {
	case *ast.TruncateTableStmt:
		err = e.executeTruncateTable(x)
	case *ast.CreateDatabaseStmt:
		err = e.executeCreateDatabase(x)
	case *ast.CreateTableStmt:
		err = e.executeCreateTable(x)
	case *ast.CreateViewStmt:
		err = e.executeCreateView(x)
	case *ast.CreateIndexStmt:
		err = e.executeCreateIndex(x)
	case *ast.DropDatabaseStmt:
		err = e.executeDropDatabase(x)
	case *ast.DropTableStmt:
		err = e.executeDropTable(x)
```

Reply:
We could keep DropTable interface intact. The DropTable and DropView could share a function named `dropTableOrVIew(ctx sessionctx.Context, tableIdent ast.Ident, isView bool)`.

Reply:
@shenli so you mean,I just rename my implement `func (d *ddl) DropTable(ctx sessionctx.Context, ti ast.Ident, isView bool) (err error) `  to `dropTableOrVIew(ctx sessionctx.Context, tableIdent ast.Ident, isView bool)` and change top interface function DropTable?